tests/test-sha1.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+const test = require("tape");
+const getSha1 = require("../sha1-utils");
+
+
+function isHexString(hashDigest) {
+  for (const character of hashDigest) {
+    if (parseInt(character, 16).toString(16) != character.toLowerCase()) {
+      return false;
+    }
+  }
+  return true;
+}
+
+test("getSha1 returns hex digest", function (t) {
+  t.plan(1);
+  t.ok(isHexString(getSha1("test@test.com")));
```

Comment:
Do we want to assert the returned value from `getSha1()` here matches an expected value for "test@test.com" (versus just checking each character is a valid hex string? Or are we doing that elsewhere?